src/sentry/api/endpoints/project_transaction_threshold.py

```diff
@@ -66,15 +79,28 @@ def post(self, request, project):
         data = serializer.validated_data
 
         with transaction.atomic():
-            project_threshold, created = ProjectTransactionThreshold.objects.update_or_create(
-                project=project,
-                organization=project.organization,
-                defaults={
-                    "threshold": data["threshold"],
-                    "metric": data["metric"],
-                    "edited_by": request.user,
-                },
-            )
+            try:
```

Comment:
Since you are changing some of the endpoint code does this merit changing any of the pytests?

Reply:
addressed in 21562668a10173794883b3dfd5fe649ad535be1d